services/gps.md

```diff
@@ -0,0 +1,58 @@
+# GPS
+
+    identifier: 0x19dd6136
+    extends: _sensor
+    camel: gps
```

Comment:
GPS is a particular American system. Should we have something more neutral and say which system we're using (GPS, GLONASS, Galileo, and the Chinese one). There are some chips that  can do several at once improving lock speed.

Reply:
Navigation system?

Reply:
positioning system; or maybe "satnav"?  (this is how the Brits call it)